[nativert] oss subgraph rewriter

dolpm opened the pull request:
Summary: att

Test Plan:
ci

Rollback Plan:

Differential Revision: D80367765

SherlockNoMad reviewed (approved):
I assume code move only, no feature change.

dolpm committed:
[nativert] oss subgraph rewriter (#160780)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/160780

att

Test Plan:
ci

Rollback Plan:

Reviewed By: SherlockNoMad, georgiaphillips

Differential Revision: D80367765
